tests/cache/failing_cache.py

```diff
@@ -2,6 +2,5 @@
 
 
 class CacheClass(LocMemCache):
-
```

Comment:
Please revert unnecessary whitespace changes.

Reply:
> Please revert unnecessary whitespace changes.

i have a problem with black and black-docs that i've outlined [here](https://forum.djangoproject.com/t/title-recurring-issue-with-black-and-blacken-docs-while-contributing-to-django/28730). i'm looking for. any pointers to help me understand more about the problem will be most welcome.

Reply:
It looks like black has stopped caring whether this blank line exists or not.

You'll need to manually revert this. (The change won't recur as long as you're using the correct version.)